src/Spago/Templates.hs

```diff
@@ -25,3 +25,15 @@ gitignore = $(embedFileUtf8 "templates/gitignore")
 
 bowerJson :: B.ByteString
 bowerJson = $(embedFile "templates/bower.json")
+
+docsSearchApp :: T.Text
+docsSearchApp =
+  $(embedURLWithFallback
+    "https://github.com/spacchetti/purescript-docs-search/releases/download/v0.0.4/docs-search-app.js"
```

Comment:
Let's factor out the tag in a top level definition, as:
- we might want to use it (e.g. in `spago version`)
- it's easier to upgrade the templates

Reply:
> ‘docsSearchVersion’ is used in a top-level splice, quasi-quote, or annotation,
>        and must be imported, not defined locally

To which module should I put it?

Reply:
@klntsky how about `Prelude`?